content/pages/community.markdown

```diff
@@ -0,0 +1,35 @@
+Title: Community
+url: community/
+save_as: community/index.html
+
+# Community
```

Comment:
The text of the markdown variant is not exactly the same as the text currently online. Make sure it is.